### PR TITLE
[Work in Progress] Codebase refactor

### PR DESCRIPTION
**Pull request by regularenthropy**
The current codebase looks messy, so I've decided to improve it.
Please, do not merge until it's finished

### CODE REVIEW

File: browser/base/content/browser-command.js [hunk 1]
```diff
@@ -441,18 +367,17 @@ function changeMuteStatus() {
...
-  if (document.getElementById("sidebar2style")){document.getElementById("sidebar2style").remove()}
-  var Tag = document.createElement("style");
-  Tag.innerText = `#webpanel{max-height:0 !important;}#sidebar2-reload,#sidebar2-forward,#sidebar2-back{display:none !important;}`
-  document.getElementsByTagName("head")[0].insertAdjacentElement('beforeend', Tag);
-  Tag.setAttribute("id", "sidebar2style");
+/* Sets the status of webpanel */
+function changeBrowserManagerPanelStatus(status){
+	if (document.getElementById("sidebar2style")){document.getElementById("sidebar2style").remove()}
+	var Tag = document.createElement("style");
+	if (status == "webpanels"){
```
Comment (regularentropy) [inline browser/base/content/browser-command.js:374]: Better to use numbers in here, but let it be like this for now.

**Comment by surapunoyousei** [comment]
How long will this pull request take? I don't have much time so I would like to know

**Comment by regularenthropy** [comment]
> How long will this pull request take? I don't have much time so I would like to know
First and second commit can already be merged. 
I've decided, that instead of doing one big refactor, it would be better to split it in a few small parts

**Comment by regularenthropy** [comment]
So. I'll col

**Comment by surapunoyousei** [comment]
Merge with Git on your local environment. There is something I would like to do to rework a little more

**Comment by regularenthropy** [comment]
> Merge with Git on your local environment. There is something I would like to do to rework a little more

Yes. I had tried it, but still, it's too big.
Furthermore, I would like to spend more time creating a new tab bar management system because the current one ([7b60ad2](https://github.com/Floorp-Projects/Floorp/pull/62/commits/7b60ad22094ede3b97358e1360ba9ed0ee4944b6) isn't dynamic enough.

**Comment by surapunoyousei** [comment]
> > Merge with Git on your local environment. There is something I would like to do to rework a little more
> 
> Yes. I had tried it, but still, the current codebase is too big. Furthermore, I would like to spend more time creating a new tab bar management system because the current one ([7b60ad2](https://github.com/Floorp-Projects/Floorp/pull/62/commits/7b60ad22094ede3b97358e1360ba9ed0ee4944b6) isn't dynamic enough.

That helps a lot

**Comment by regularenthropy** [comment]
> > > Merge with Git on your local environment. There is something I would like to do to rework a little more
> > 
> > 
> > Yes. I had tried it, but still, the current codebase is too big. Furthermore, I would like to spend more time creating a new tab bar management system because the current one ([7b60ad2](https://github.com/Floorp-Projects/Floorp/pull/62/commits/7b60ad22094ede3b97358e1360ba9ed0ee4944b6) isn't dynamic enough.
> 
> That helps a lot

Great! I'll open a PR for the 1st and 2nd commit.

**Comment by surapunoyousei** [comment]
I see

**Pull request closed without merging**